pkg/kafka/auth/config.go

```diff
@@ -79,15 +80,14 @@ func (config *AuthenticationConfig) InitFromViper(configPrefix string, v *viper.
 		Prefix: configPrefix,
 	}
 
-	var err error
-	config.TLS, err = tlsClientConfig.InitFromViper(v)
+	opts, err := tlsClientConfig.InitFromViper(v)
```

Comment:
nit: poor readability
```suggestion
	tlsOpts, err := tlsClientConfig.InitFromViper(v)
```